.github/workflows/performance-tests.yml

```
@@ -0,0 +1,530 @@
+        with:
+          name: performance-baselines
+          path: reports/performance-baselines/
+          retention-days: 90
+
+  # Analysis and reporting job
+  analyze-results:
+    name: Analyze Results
+    runs-on: ubuntu-latest
+    needs: [setup, performance-tests]
+    if: always() && needs.performance-tests.result != 'skipped'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)